src/computed/wallet.js

```diff
@@ -21,8 +21,14 @@ const ComputedWallet = store => {
         balanceSatoshis,
         pendingBalanceSatoshis,
         channelBalanceSatoshis,
+        limboBalanceSatoshis,
       } = store;
-      return balanceSatoshis + pendingBalanceSatoshis + channelBalanceSatoshis;
+      return (
+        balanceSatoshis +
+        pendingBalanceSatoshis +
+        channelBalanceSatoshis +
+        limboBalanceSatoshis
+      );
```

Comment:
Although `store.limboBalanceSatoshis` is set in channel actions I would also compute this here 👍

Reply:
Sorry, I'm confused what this means. What is being computed?

Reply:
Just saying it makes sense to use the `ComputedWallet` module here. So no changes needed :)